packages/create-plugin/templates/datasource/src/components/ConfigEditor.tsx

```diff
@@ -39,29 +43,29 @@ export function ConfigEditor(props: Props) {
     });
   };
 
-  const { jsonData, secureJsonFields } = options;
-  const secureJsonData = (options.secureJsonData || {}) as MySecureJsonData;
-
   return (
-    <div className="gf-form-group">
-      <InlineField label="Path" labelWidth={12}>
+    <>
+      <InlineField label="Path" labelWidth={14} interactive tooltip={'Json field returned to frontend'}>
```

Comment:
Re label width: Opened a [PR](https://github.com/grafana/grafana/pull/84970) in Grafana to increase the width of the name field to 14. That way we at least give the plugin authors a chance to align fields width across the entire page.